src/Dapr.PluggableComponents/Components/SecretStores/SecretStoreBulkGetResponse.cs

```diff
@@ -0,0 +1,37 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2023 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using Dapr.PluggableComponents.Utilities;
+using Dapr.Proto.Components.V1;
+using Google.Protobuf;
+
+namespace Dapr.PluggableComponents.Components.SecretStores;
+
+public sealed record SecretStoreBulkGetResponse
+{
+    //public IReadOnlyDictionary<string, Dictionary<string, string>> responseData = new Dictionary<string, Dictionary<string, string>>();
```

Comment:
`IReadOnlyDictionary<string, IReadonlyDictionary<string, string>>` would be the most *raw* way to expose the bulk data, but I'm not sure if that'd be the *best* way, as it doesn't easily allow for changes to the API in the future.  (E.g. suppose the bulk request returns both values and other metadata for each individual secret).  It'd probably be better to expose something like `IReadOnlyDictionary<string, GetSecretResponse> Data` (which is actually closer to the `.proto` definition anyway).

I think it's fine to try to simplify some of the `.proto` APIs where it makes sense (e.g. where it contains unnecessary layers), but we also want to walk that fine line of being true to the `.proto` as well so that the .NET APIs are still conceptually similar to the other SDKs.

Reply:
I think we can use the `SecretResponse()` constructor from the proto defintion.